### PR TITLE
Ideanote (Certified Connector)

### PR DESCRIPTION
**Pull request by runem**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

### CODE REVIEW

File: certified-connectors/Ideanote/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+## Ideanote Connector
+
+Ideanote is the next-generation innovation software that has everything you need to collect, develop, prioritize and act on more of the right ideas.
+
```
Comment (sriyen-msft) [inline certified-connectors/Ideanote/README.md:4]: @runem Please find a template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/readme.md) and add the other sections.

File: certified-connectors/Ideanote/apiProperties.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+	"properties": {
+		"connectionParameters": {
```
Comment (sriyen-msft) [inline certified-connectors/Ideanote/apiProperties.json:3]: @runem Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa's%20Talking%20Airtime/apiProperties.json) for reference.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Ideanote/README.md (hunk 1 above)
Comment (sriyen-msft) [reply certified-connectors/Ideanote/README.md:3]: @runem Please add the required sections such as "Publisher". I have already shared the template in my previous comment.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@runem Please add the required sections such as "Publisher". I have already shared the template in my previous comment.

File: certified-connectors/Ideanote/README.md (hunk 1 above)
Comment (runem) [reply certified-connectors/Ideanote/README.md:5]: @sriyen-msft Sorry, I misread the publisher section as being only required for independent publishers. The section has been added now.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**